go-controller/pkg/ovn/base_network_controller_kubevirt.go

```diff
@@ -0,0 +1,63 @@
+package ovn
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+)
+
+func (bnc *BaseNetworkController) ensureNetworkInfoForVM(origPod *corev1.Pod) error {
+	if !kubevirt.PodIsLiveMigratable(origPod) {
+		return nil
+	}
+	vmNetworkInfo, err := kubevirt.FindNetworkInfo(bnc.watchFactory, origPod)
+	if err != nil {
+		return err
+	}
+	resultErr := retry.RetryOnConflict(util.OvnConflictBackoff, func() error {
+		// Informer cache should not be mutated, so get a copy of the object
+		pod, err := bnc.watchFactory.GetPod(origPod.Namespace, origPod.Name)
+		if err != nil {
+			return err
+		}
+
+		cpod := pod.DeepCopy()
+		_, ok := cpod.Labels[kubevirt.OriginalSwitchNameLabel]
+		if !ok {
+			cpod.Labels[kubevirt.OriginalSwitchNameLabel] = vmNetworkInfo.OriginalSwitchName
+		}
+		if vmNetworkInfo.Status != "" {
+			cpod.Annotations[util.OvnPodAnnotationName] = vmNetworkInfo.Status
+		}
+		return bnc.kube.UpdatePod(cpod)
+	})
+	if resultErr != nil {
+		return fmt.Errorf("failed to update labels and annotations on pod %s/%s: %v", origPod.Namespace, origPod.Name, resultErr)
+	}
+
+	// There is nothing to check
+	if vmNetworkInfo.Status == "" {
+		return nil
```

Comment:
This means that the controller didn't assign an annotation yet, the pod will be reconciled since it's impossible to get the switch subnet there are other checks after this so another warning is adding more noise.

Reply:
OK got it.